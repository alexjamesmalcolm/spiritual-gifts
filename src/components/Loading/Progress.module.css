.container {
  width: 100%;
  height: 100%;
  position: absolute;
  display: flex;
  justify-content: center;
  align-items: center;
}

.progress {
  width: 80%;
  height: 5rem;
  margin: auto;
}
