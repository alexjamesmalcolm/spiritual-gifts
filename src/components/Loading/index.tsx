import React from "react";
import styles from "./Loading.module.css";

const Loading = () => (
  <div className={styles.container}>
    <div className={styles.loader} />
  </div>
);

export default Loading;
