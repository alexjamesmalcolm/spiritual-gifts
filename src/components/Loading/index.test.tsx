import { render } from "@testing-library/react";
import Loading from ".";

describe("Loading", () => {
  it("should render spinner", () => {
    const { container } = render(<Loading />);
    expect(container).toMatchSnapshot();
  });
});
