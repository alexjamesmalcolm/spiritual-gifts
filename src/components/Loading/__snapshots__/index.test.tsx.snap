// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading should render spinner 1`] = `
<div>
  <div
    class="container"
  >
    <div
      class="loader"
    />
  </div>
</div>
`;
